### PR TITLE
added documentation for the logging

### PR DESCRIPTION
**Pull request by xmairaa**
Added logging documentation
page size increased 640px to 1200px, for better view
Little bit modifications on documentation

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -670,6 +670,25 @@ <h3>
 ]
 </code></pre>
 <p>You can open the RabbitMQ's management console and find these messages in a queue.</p>
+
+<h2>Logging</h2>
+
+
+<p>REMReM publish application logging is implemented using the logback-classic, and we required to configure the logback.xml file, <br/>
```
Comment (ValentinTyhonov) [inline index.html]: Remove comma at the end of line and add dot if it is needed.
REMReM Publish instead of REMReM publish like in generate documentation

**Comment by ValentinTyhonov** [comment]
Looks good for me

File: index.html [hunk 2]
```diff
@@ -62,9 +62,9 @@ <h2>
 <h2>
 <a id="how-to-install" class="anchor" href="#how-to-install" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>How to Install?</h2>
 
-<p>Binary is relased via <a href="https://jitpack.io/#Ericsson/eiffel-remrem-publish">jitPack</a></p>
-<p><b>Download the latest REMReM Publish CLI Binary via </b><a onClick="getLatestVersion('cli','jar');">publish-cli.jar</a></p>
-<p><b>Download the latest REMReM Publish Service Binary via </b><a onClick="getLatestVersion('service','war');">publish-service.war</a></p>
+<p>Binary is release via <a href="https://jitpack.io/#Ericsson/eiffel-remrem-publish">jitPack</a></p>
```
Comment (xvinosi-github) [inline index.html]: released

File: index.html [hunk 3]
```diff
@@ -670,6 +670,25 @@ <h3>
 ]
 </code></pre>
 <p>You can open the RabbitMQ's management console and find these messages in a queue.</p>
+
+<h2>Logging</h2>
+
+
+<p>REMReM Publish application logging is implemented using the logback-classic, and we required to configure the logback.xml file. <br/>
```
Comment (xvinosi-github) [inline index.html]: logback-classic. It requires user to configure the logback.xml file.
When using without logback.xml log messages will look as below:
To configure the logback.xml file use  <strong>-Dlogging.config=path/logback.xml</strong>


To get sample logback.xml to use Click Here

File: index.html [hunk 4]
```diff
@@ -124,10 +124,10 @@ <h2>
...
  -h,--help                      show help.
- -json,--json_content &lt;arg&gt;     event content in JSON string. The value can
-                                also be a dash(-) and the JSON will be read
+ -json,--json_content &lt;arg&gt;     event content in json string. The value can
+                                also be a dash(-) and the json will be read
                                 from the output of other programs if piped. Multiple
-                                JSON events are also supported.
+                                json events are also supported.
  -mb,--message_bus &lt;arg&gt;        host of message bus to use
  -mp,--message_protocol &lt;arg&gt;   name of messaging protocol to be used, e.g. eiffel3,
```
Comment (xvinosi-github) [inline index.html]: we could remove eiffel3 and put let say myeiffelprotocol

File: index.html [hunk 5]
```diff
@@ -674,18 +674,18 @@ <h3>
 <h2>Logging</h2>
 
 
-<p>REMReM Publish application logging is implemented using the logback-classic, and we required to configure the logback.xml file. <br/>
-   With out logback.xml configuration log will get the PARSER_ERROR like below
+<p>REMReM Publish application logging is implemented using the logback-classic. It requires user to configure the logback.xml file.<br/>
+When using without logback.xml log messages will look as below: <br/>
```
Comment (xvinosi-github) [inline index.html]: When used

**Pull request merged**